build/resources/main/templates/register.html

```diff
@@ -3,52 +3,30 @@
 <head th:replace="~{fragments :: head}"></head>
 <body>
 <header th:replace="~{fragments :: header}"></header>
-<main>
-    <div class="register text-center">
+    <main>
+        <div class="register">
```

Comment:
Same comment as above - just need to keep the newest version of register.html 
e.g starting at line 7 it says:   <div class="register text-center">